src/control/system/database_members.go

```diff
@@ -68,6 +68,23 @@ func (mam MemberAddrMap) addMember(addr *net.TCPAddr, m *Member) {
 	mam[addr.String()] = append(mam[addr.String()], m)
 }
 
+func (mam MemberAddrMap) removeMember(m *Member) {
+	members, ok := mam[m.Addr.String()]
+	if !ok {
+		return
+	}
+	for i, mem := range members {
+		if m.UUID.String() == mem.UUID.String() {
```

Comment:
might be nicer to have m.UUID.Equals() method

Reply:
Agreed. This is a uuid.UUID, though, and I didn't see an existing comparison function.

Reply:
For the record, these UUIDs are directly comparable because they are byte arrays rather than slices. From https://golang.org/ref/spec#Comparison_operators:
> Array values are comparable if values of the array element type are comparable. Two array values are equal if their corresponding elements are equal. 

If you look at the unit tests for the package, e.g. https://github.com/google/uuid/blob/master/uuid_test.go#L120 you can see that they're doing direct comparison.

Reply:
Oh nice. Want that corrected now? Or I can get it in a follow-on, if there's nothing else to be changed.

Reply:
That's fine with me

Reply:
Fixed